docs/troubleshoot/app-framework/zss-error-codes.md

```diff
@@ -0,0 +1,371 @@
+# ZSS Error Message Codes
+
+The following error message codes may appear on ZSS log. Use the following message code references and the corresponding reasons and actions to help troubleshoot issues.
+
+## ZSS informational messages
+
+### ZWES0013I
+
+  ZSS Server has started. Version '%s'
+
+  **Reason:**
+
+  ZSS Server has started.
+
+  **Action:**
+
+  No action required.
+
+### ZWES0014I
+
+  ZIS status - '%s' (name='%.16s', cmsRC='%d', description='%s', clientVersion='%d')
+
+  **Reason:**
+
+  The message shows status of the connection to Privileged Server: ZIS status - `<OK or Failure>` (name=`<Privileged Server Name>`, cmsRC=`<RC>`, description=`<description`, clientVersion=`<version>`)
```

Comment:
Do we have a list of RC and descriptions?  If not, we should make a ticket to track getting these as well, since they come up often.

Reply:
Most of the codes are meaningful only for developers so I documented only those that important for users/admins.